src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.product-card__reviews-stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.product-card__reviews-stars--1 :nth-child(1),
+.product-card__reviews-stars--2 :nth-child(-n + 2),
+.product-card__reviews-stars--3 :nth-child(-n + 3),
+.product-card__reviews-stars--4 :nth-child(-n + 4),
+.product-card__reviews-stars--5 :nth-child(-n + 5) {
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), the modifier should be separated by double underscores instead of a single dash. Please update the class names accordingly to follow the BEM naming convention.